subprojects/core-api/src/main/java/org/gradle/api/artifacts/transform/TransformParameters.java

```diff
@@ -39,7 +39,6 @@
  *
  * @since 5.3
  */
-@Incubating
```

Comment:
I think the `None` subtype should also be de-incubated.

Reply:
Done.